### PR TITLE
Add note about `importMap` option in config file

### PR DESCRIPTION
**Pull request by kidonng**
Adding this to the manual so others don't have to read the JSON schema of config file in order to realize the existence of `importMap` option like me.

Also improved the links in Introduction page for GitHub a little bit.

### CODE REVIEW

File: introduction.md [hunk 1]
```diff
@@ -13,12 +13,12 @@ It's built on V8, Rust, and Tokio.
...
-- Supports [TypeScript](./typescript/) out of the box.
+- Supports [TypeScript](./typescript.md) out of the box.
 - Ships a single executable (`deno`).
-- Provides built-in [development tooling](./tools) like a code formatter
+- Provides built-in [development tooling](./tools.md) like a code formatter
   ([`deno fmt`](./tools/formatter.md)), a linter
-  ([`deno lint`](./tools/linter.md)), a test runner ([`deno test`](./testing)),
-  and a
+  ([`deno lint`](./tools/linter.md)), a test runner
+  ([`deno test`](./testing.md)), and a
```
Comment (bartlomieju) [inline introduction.md:21]: These are unrelated change, please remove

**Review by bartlomieju — CHANGES_REQUESTED** [review]

File: introduction.md (hunk 1 above)
Comment (kidonng) [reply introduction.md]: These changes link to the correct file on GitHub, but since you requested, I will put them in another PR.

**Review by bartlomieju — APPROVED** [review]
Thanks!
